common/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
     compileOnly group: 'org.apache.commons', name: 'commons-text', version: '1.10.0'
     compileOnly group: 'com.google.code.gson', name: 'gson', version: '2.10.1'
     compileOnly group: 'org.json', name: 'json', version: '20231013'
+    testImplementation group: 'org.json', name: 'json', version: '20231013'
```

Comment:
implementation group: 'org.json', name: 'json', version: '20231013'

I think this serves as both line 28 + 29 so no needs to declare twice here for the same dependency. Also if you want to go the extra mile, using "api group: 'org.json', name: 'json', version: '20231013'" can remove the duplicate of this dependency in other modules.

Reply:
Fair enough and fine with this new line. But if you check this (https://github.com/opensearch-project/ml-commons/blob/main/ml-algorithms/build.gradle#L76), it's already added in runtime in several module too.

Reply:
Ref PR: https://github.com/opensearch-project/ml-commons/pull/1062/files